pxr/usd/usdGeom/mesh.h

```diff
@@ -231,15 +272,9 @@ class UsdGeomMesh : public UsdGeomPointBased
     // --------------------------------------------------------------------- //
     /// The subdivision scheme to be applied to the surface.  
     /// Valid values are "catmullClark" (the default), "loop", "bilinear", and
-    /// "none" (i.e. a polymesh with no subdivision - the primary difference
-    /// between schemes "bilinear" and "none" is that bilinearly subdivided
-    /// meshes can be considered watertight, whereas there is no such guarantee
-    /// for un-subdivided polymeshes, and more mesh features (e.g. holes) may
-    /// apply to bilinear meshes but not polymeshes.   Further, if a polymesh 
-    /// does not provide normals, then it should be treated and rendered as 
-    /// faceted, whereas a "bilinear" Mesh should compute smooth normals. 
-    /// For primarily this latter reason, polymeshes \em may be lighterweight 
-    /// and faster to render, depending on renderer and render mode.)
+    /// "none" (i.e. a polygonal mesh with no subdivision).  Polygonal meshes
+    /// are typically lighterweight and faster to render, depending on renderer
+    /// and render mode.
```

Comment:
I agree that we don't need to make a big distinction about it, but I do feel like there's something missing if we don't say anything at all about the distinction between polymeshes and bilinear. Perhaps one way to resolve this is to list the options in a list like we do above for interpolations, and below for face varying linear interpolation modes?

Reply:
Agreed, it is a bit too lean.  I like the idea of the list, but "bilinear" will precede "none" in that list, and contrasting the two seems seems more awkward as a result.  How about appending the sentence:   "Use of "bilinear" will produce a similar shape to a polygonal mesh and may offer additional guarantees of watertightness and additional subdivision features (e.g. holes) but may also not respect authored normals."

Reply:
I actually expanded "subdivisionScheme" to the list you suggested for other reasons (now in recent commit) and appended the sentence noted here.